programs/issue-3-speed-converter/run.py

```diff
@@ -0,0 +1,76 @@
+convert_dictionary = {
+ 1: {
+     "input": "m/s",
+     "output": "km/h",
+     "equation": "value * 3.6"},
+ 2: {
+     "input": "km/h",
+     "output": "m/s",
+     "equation": "value / 3.6"},
+ 3: {
+     "input": "mile/h",
+     "output": "km/h",
+     "equation": "value * 1.609"},
+ 4:  {
+     "input": "km/h",
+     "output": "mile/h",
+     "equation": "value / 1.609"},
+ 5: {
+     "input": "km/h",
+     "output": "% prędkości światła",
+     "equation": "value / 1.079e+9"}
+}
+
+
+def converter(value, options):
+    try:
+        input, output, equation = options["input"], options["output"], options["equation"]
+    except KeyError:
+        print("Coś poszło nie tak. Spróbuj ponownie później.")
+        return
+    converted = eval(equation)
```

Comment:
using eval is really, Really, REALLY a bad practice: 

https://youtu.be/aEj8W_K1IFE?t=1861